### DISCUSSION TITLE
is it possible to join two standalone machines, if yes, how?

### DISCUSSION THREAD

**Discussion by rock-kobs** [Security-Onion-Solutions/securityonion#7824] [General] [answered]
i have a test lab which is standalone-config and another one which i don't mess with also standalone-config just wondering if its possible to join the two machines like you do during dis-setup, am looking for a command like so-remote-join test lab to the more serious one if you like. thanks in advance. if its not possible and the only solution is to setup again and choose to add the sensor for example to the main machine can we have this option available for standalone setup as well. for example when you choose standalone you get the option to either join it to another manager or whatever other setup one may have. thank you.

**Comment by dougburks** [comment, marked as answer]
Joining two standalone machines together is not supported at this time. 

The supported options for joining multiple machines together into one grid can be seen here:
https://docs.securityonion.net/en/2.3/architecture.html#distributed
